README.md

```diff
@@ -27,7 +27,7 @@ The Utility Connect widget can be instantiated for one of two user flows:
 
```

Comment:
At the end of list item one I would add "('create' mode)" and at the end of list item two I would add "('update' mode)" so that when we make references to those modes further down, it's not confusing.